entrypoint.sh

```diff
@@ -36,7 +36,11 @@ fi
 
 # Deploy the Fly app, creating it first if needed.
 if ! flyctl status --app "$app"; then
-  flyctl launch --now --copy-config --name "$app" --image "$image" --region "$region" --org "$org"
+  flyctl launch --no-deploy --copy-config --name "$app" --image "$image" --region "$region" --org "$org"
+  if [ -n "$INPUT_SECRETS" ]; then
+    echo $INPUT_SECRETS | tr " " "\n" | flyctl secrets import --app "$app"
```

Comment:
Wouldn't this break with secrets that contain spaces?

Reply:
It would, but I think that's ok for now. I'm going to go ahead and merge this and we can make the new option more resilient on a second pass.

Reply:
Hi, I set the secret like so but it doesn't work for me. Is it a correct way to set the secret?
```
 secrets: SANITY_DATASET=staging
```

Reply:
@mrkurt My secret has spaces and the action doesn't work

```
GOOGLE_PRIVATE_KEY=-----BEGIN PRIVATE KEY-----\nxxx\n-----END PRIVATE KEY-----\n
```

Is there any walkaround?